blacktie/integration-tests/src/test/java/org/jboss/narayana/blacktie/jatmibroker/xatmi/CSControl.java

```diff
@@ -53,10 +54,14 @@ public void tearDown() {
         try {
             if (server != null) {
                 log.debug("destroying server process");
-                server.interrupt();
-                server.getProcess().destroy();
-                log.debug("destroyed server process");
+                try {
+                    int pid = getUnixPID(server.getProcess());
+                    Runtime.getRuntime().exec("kill -INT " + pid).waitFor();
```

Comment:
Ah, can cs.exe propagate the interrupt then? Trap the interrupt and send it to the server.exe? 

The issue with the current fix is it won't work on Windows. I realise its a test issue but it could happen on windows too.